.gitignore

```diff
@@ -4,6 +4,7 @@
 .eslintcache
 .DS_Store
 *.pem
+.idea
```

Comment:
nitpick: might be worth adding to your global `.gitignore` rather than here, but doesn't *really* matter either way :)

Reply:
I'll leave it here but I also added to my global one.